index.bs

```diff
@@ -355,6 +371,53 @@ or failure:
 	append a duplicate of the second element to |tokens|.
 7. Return |tokens|.
 
+The <dfn for="IntersectionObserver">scrollable intersection rectangles</dfn>
```

Comment:
Can we get rid of this definition and use the existing specified term `scrollport` where necessary?

https://www.w3.org/TR/css-overflow-3/#scrollport

Reply:
Good idea.